aries-site/src/pages/design-tokens/how-to-read-design-tokens.mdx

```diff
@@ -0,0 +1,100 @@
+
+## Types of design tokens
+
+Each type of token uses a subset of the overall convention. To learn about types of tokens, read the documentation of [types of design tokens](/design-tokens/overview#types-of-design-tokens).
+
+### Primitive
+
+<Example height="auto">
+  <TokenHighlight highlight={['prefix', 'level', 'type', 'variant', 'scale']} />
+</Example>
+
+| prefix | level | type      | variant | scale | Token                    |
+| ------ | ----- | --------- | ------- | ----- | ------------------------ |
+| hpe    | base  | color     | green   | 100   | hpe.base.color.green.100 |
+| hpe    | base  | dimension | --      | 400   | hpe.base.dimension.400   |
+
+### Semantic
+
+Semantic tokens represent global patterns.
+
+<Example height="auto">
+  <TokenHighlight
+    highlight={['prefix', 'type', 'variant', 'element', 'scale', 'state']}
+  />
+</Example>
+
+| prefix | type    | variant    | state    | scale  | Token                              |
+| ------ | ------- | ---------- | -------- | ------ | ---------------------------------- |
+| hpe    | color   | primary    | --       | strong | hpe.color.primary.strong           |
+| hpe    | spacing | --         | --       | large  | hpe.spacing.large                  |
+| hpe    | color   | background | selected | weak   | hpe.color.background.selected.weak |
+
+### Component
+
+Component tokens represent tokens for specific components and should not be used to style a different component.
+
+<Example height="auto">
+  <TokenHighlight
+    highlight={[
+      'prefix',
+      'component',
+      'size',
+      'element',
+      'variant',
+      'state',
+      'property',
+    ]}
+  />
+</Example>
+
+| prefix | component | size   | element | variant   | state    | property    | Token                                   |
+| ------ | --------- | ------ | ------- | --------- | -------- | ----------- | --------------------------------------- |
+| hpe    | button    | medium | --      | secondary | --       | borderWidth | hpe.button.medium.secondary.borderWidth |
+| hpe    | button    | --     | --      | primary   | disabled | background  | hpe.button.primary.disabled.background  |
+| hpe    | menu      | medium | item    | --        | --       | paddingX    | hpe.menu.medium.item.paddingX           |
+| hpe    | menu      | --     | item    | --        | enabled  | background  | hpe.menu.item.enabled.background        |
+
+## Token delimiters
+
+While our documentation uses `.` dot as a delimiter, token parts should be separated by the appropriate character for a given framework. For example, CSS variables use the `-` character.
```

Comment:
I think we try to avoid possessives (e.g. our, your), but the pros/cons are something we can discuss as a team. cc: @julauxen , @oliverHPE .

```suggestion
While this documentation uses `.` dot as a delimiter, token parts should be separated by the appropriate character for a given framework. For example, CSS variables use the `-` character.
```

Reply:
Done.